File: Models/Character.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace taleOfDungir.Models
{
    public class Character
    {
        public Int64 CharacterId { get; set; }
        
        public Int64 Gold { get; set; }
        public int Level { get; set; }
        public Int64 Exp { get; set; }

        //Stats, WORK_IN_PROGRESS
        public Int64 Health { get; set; }
        //

        public List<Item> Inventory { get; set; }//Items in bag
        public List<Item> Equipment { get; set; }//Equipped items

        public Skills Skills { get; set; }//Combat skills
        public LifeSkills LifeSkills { get; set; }//Event skills

        public string ApplicationUserId { get; set; }
        public ApplicationUser ApplicationUser { get; set; }
    }
}

File: ClientApp/src/_models/Character.ts
import { Equipment } from './Equipment';
import { Entity } from './Entity';
import { Item } from './Item';
import { LifeSkills } from './LifeSkills';
import { Skills } from './Skills';
export class Character {
    exp: number;
    reqExp: number;
    gold: number;
    level: number;
    skills: Skills;
    lifeSkills: LifeSkills;
    inventory?: Item[];
    equipment?: Equipment;
    entity?: Entity;
}